proto/viam/component/camera/v1/camera.proto

```diff
@@ -109,17 +109,6 @@ message Webcam {
 }
 
 message Property {
-  // Camera video properties
-  Video video = 1;
-  // Video resolution width
-  int32 width = 2;
-  // Video resolution height
-  int32 height = 3;
-  // Video frame format
-  string frame_format = 4;
-}
-
-message Video {
```

Comment:
sorry, this diff is confusing - i'm basically removing the `video` field, which contains the same fields as what remains in the `Property` struct